pkg/util/test/util.go

```diff
@@ -428,15 +428,16 @@ func validateStorageNodes(pxImageList map[string]string, cluster *corev1.Storage
 		if len(cluster.Spec.Env) > 0 {
 			for _, env := range cluster.Spec.Env {
 				if env.Name == PxReleaseManifestURLEnvVarName {
-					pxVersion = strings.TrimSpace(regexp.MustCompile(`\S+\/(\S+)\/version`).FindStringSubmatch(env.Value)[1])
-					if pxVersion == "" {
-						return fmt.Errorf("failed to extract version from value of %s", PxReleaseManifestURLEnvVarName)
-					}
+					expectedPxVersion = strings.TrimSpace(regexp.MustCompile(`\S+\/(\S+)\/version`).FindStringSubmatch(env.Value)[1])
```

Comment:
How do we handle master where the url does not have the version?

Reply:
We haven't before, added support for handling master builds